plugins/woocommerce-admin/client/task-lists/fills/products/constants.tsx

```diff
@@ -90,3 +90,6 @@ export const defaultSurfacedProductTypes =
 	onboardingProductTypesToSurfaced.physical;
 
 export const supportedOnboardingProductTypes = [ 'physical', 'downloads' ];
+
+export const SETUP_TASKLIST_PRODUCT_TYPES =
```

Comment:
Constant name could be a tad clearer. What do you think?
```suggestion
export const SETUP_TASKLIST_PRODUCT_TYPES_FILTER =
```